### PR TITLE
Update connection instead of deletion if connection exists

### PR DESCRIPTION
**Pull request by youngyjd**
Let's do not delete old connections if we are going to update for a connection. There may be a race condition that customer's job may will fail during deployment because of connection not found.

### CODE REVIEW

**Comment by youngyjd** [comment]
@astahlman @sherryzhao1006

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -1421,40 +1421,33 @@ def connections(args):
     if args.filepath:
         with open(args.filepath, 'r') as stream:
             connections_config = yaml.load(stream)
-        connections = {
-            connection['conn_id']: connection['conn_uri']
-            for connection in connections_config['connections']
-        }
-        # check conn_id and conn_uri were passed from yaml file.
-        missing_args = list()
```
Comment (sherryzhao1006) [inline airflow/bin/cli.py:1429]: why we want to delete this check?

Comment (youngyjd) [reply airflow/bin/cli.py:1429]: I put it down here https://github.com/lyft/incubator-airflow/pull/191/files#diff-1c2404a3a60f829127232842250ff406R1432 so we just have to loop connection_config once

**Comment by youngyjd** [comment]
@astahlman Yeah. We have this module https://github.com/lyft/airflowinfra/blob/c6c43d1ad1620829740df82df3d3e02b99bd5d2f/ops/config/modules/states/airflowinfra_connection.py#L54 Credit to Sherry.

**Pull request merged**